app/api/__init__.py

```diff
@@ -102,19 +102,20 @@
 # microlocations
 api.route(MicrolocationList, 'microlocation_list', '/microlocations', '/sessions/<int:id>/microlocations')
 api.route(MicrolocationDetail, 'microlocation_detail', '/microlocations/<int:id>',
-          '/events/<int:event_id>/microlocations', '/events/<identifier>/microlocations',
+          '/events/<int:event_id>/microlocations', '/events/<event_identifier>/microlocations',
           '/sessions/<int:session_id>/microlocation')
 api.route(MicrolocationRelationship, 'microlocation_session',
           '/microlocations/<int:id>/relationships/sessions')
 api.route(MicrolocationRelationship, 'microlocation_event',
           '/microlocations/<int:id>/relationships/event')
 
 # sessions
-api.route(SessionList, 'session_list', '/sessions', '/events/<int:event_id>/sessions', '/events/<identifier>/sessions',
+api.route(SessionList, 'session_list', '/sessions', '/events/<int:id>/sessions', '/events/<identifier>/sessions',
```

Comment:
Please don't change it back to `id` as we already have to correct this to `event_id` in separate PR as well.